### PR TITLE
Issue #805 by Inlead: Added htmlmail module and hook enabled.

### PR DESCRIPTION
**Pull request by guddo**
http://platform.dandigbib.org/issues/805

### CODE REVIEW

File: ding2.make [hunk 1]
```diff
@@ -121,6 +121,9 @@ projects[globalredirect][patch][] = "http://drupal.org/files/language_redirect_v
 projects[google_analytics][subdir] = "contrib"
 projects[google_analytics][version] = "1.3"
 
+projects[htmlmail][subdir] = "contrib"
+projects[htmlmail][version] = "7.x-2.65"
```
Comment (kasperg) [inline ding2.make]: The build fails because the version declaration is invalid. It should be 2.65.

Comment (guddo) [inline ding2.make]: *doh*

Comment (kasperg) [inline ding2.make]: :+1: CircleCI :smile:

**Comment by guddo** [comment]
Moved to https://github.com/ding2/ding2/pull/234.

**Pull request closed without merging**